### PR TITLE
Montana Taxable Social Security Benefits

### LINKED ISSUE #2623: Initial Montana Taxable Social Security Benefits subtraction
https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2023/05/Montana-Idividiual-Income-Tax-Return-Form-2-2022v6.2.pdf

### PR DESCRIPTION
**Pull request by qingqiuzhang**
Fixes #2623

### Summary
:sparkles::card_file_box::test_tube:

This pull request adds a new feature to the policy engine that calculates the taxable amount of the social security benefit in Montana, based on the state-specific rules and parameters. It introduces several new parameter and variable files under the `gov/states/mt/tax/income` directory, as well as corresponding unit tests. It also updates the changelog file to reflect the minor version bump.

> _`Montana` feature_
> _Taxing social security_
> _Fall leaves are falling_

### Walkthrough
*  Add a new feature to calculate the taxable amount of the social security benefit in Montana ([link](https://github.com/PolicyEngine/policyengine-us/pull/3247/files?diff=unified&w=0#diff-5885042762833a4aef51405a4036067e4e6aefa7a5091a2987d1d9d1fc2647e8L1-R4), [link](https://github.com/PolicyEngine/policyengine-us/pull/3247/files?diff=unified&w=0#diff-fa62a4c0e260051bfe05aa19a5113c4260ff61e787e345d2223cbea302088b3fR1-R22), [link](https://github.com/PolicyEngine/policyengine-us/pull/3247/files?diff=unified&w=0#diff-33219edb927aa01e8f573b93aacfe509e832a9be53cca56b108a5231514147c0R1-R13), [link](https://github.com/PolicyEngine/policyengine-us/pull/3247/files?diff=unified&w=0#diff-4651dc1fc9cb886b36174ebfb73f4527f61283793a3268e96ef7ee586af84b24R1-R13), [link](https://github.com/PolicyEngine/policyengine-us/pull/3247/files?diff=unified&w=0#diff-ac3391c3723f25d8be490fe71f4b7451f6664b5fa6f0b5812bad264b191a7fa4R1-R13), [link](https://github.com/PolicyEngine/policyengine-us/pull/3247/files?diff=unified&w=0#diff-bed5f648b9858e5763ee90ad85315aac8a740a5e925a14a3a5ed9a5134afaea4R1-R13), [link](https://github.com/PolicyEngine/policyengine-us/pull/3247/files?diff=unified&w=0#diff-5559049c4f6b54ea1c2369578d50d1470e6eeb04100bd3daa371f16730ec7cfdR1-R23), [link](https://github.com/PolicyEngine/policyengine-us/pull/3247/files?diff=unified&w=0#diff-a079a93586ee0df2541b3aa0f4c58a07f2416ddd345cc7a827541cda9a659703R1-R25), [link](https://github.com/PolicyEngine/policyengine-us/pull/3247/files?diff=unified&w=0#diff-9af998c5ea176d3da9094fc82f22de3b7267192046a787c16381aa603e01d3d4R1-R20), [link](https://github.com/PolicyEngine/policyengine-us/pull/3247/files?diff=unified&w=0#diff-dee212690bf95de551fb341aff32d261986643c7b7332fe806a1c70a9232d9a1R1-R50))
  - Define the `mt_modified_income` variable as the income measure used for the feature, based on the federal AGI and various additions, subtractions, and adjustments ([link](https://github.com/PolicyEngine/policyengine-us/pull/3247/files?diff=unified&w=0#diff-a079a93586ee0df2541b3aa0f4c58a07f2416ddd345cc7a827541cda9a659703R1-R25))
  - Define the `mt_modified_income_benefit_fraction` variable as the amount of the total social security benefits that is multiplied by the first fraction parameter ([link](https://github.com/PolicyEngine/policyengine-us/pull/3247/files?diff=unified&w=0#diff-9af998c5ea176d3da9094fc82f22de3b7267192046a787c16381aa603e01d3d4R1-R20))
  - Define the `mt_social_security_benefit` variable as the taxable amount of the social security benefit, using the minimum of the federal taxable benefit and the alternative subtraction amount, which depends on the `mt_modified_income` variable and several parameters ([link](https://github.com/PolicyEngine/policyengine-us/pull/3247/files?diff=unified&w=0#diff-dee212690bf95de551fb341aff32d261986643c7b7332fe806a1c70a9232d9a1R1-R50))
  - Add parameter files for the base amount, the modified income cap, the exceeding income cap, and the fractions used for the alternative subtraction amount calculation ([link](https://github.com/PolicyEngine/policyengine-us/pull/3247/files?diff=unified&w=0#diff-fa62a4c0e260051bfe05aa19a5113c4260ff61e787e345d2223cbea302088b3fR1-R22), [link](https://github.com/PolicyEngine/policyengine-us/pull/3247/files?diff=unified&w=0#diff-33219edb927aa01e8f573b93aacfe509e832a9be53cca56b108a5231514147c0R1-R13), [link](https://github.com/PolicyEngine/policyengine-us/pull/3247/files?diff=unified&w=0#diff-4651dc1fc9cb886b36174ebfb73f4527f61283793a3268e96ef7ee586af84b24R1-R13), [link](https://github.com/PolicyEngine/policyengine-us/pull/3247/files?diff=unified&w=0#diff-ac3391c3723f25d8be490fe71f4b7451f6664b5fa6f0b5812bad264b191a7fa4R1-R13), [link](https://github.com/PolicyEngine/policyengine-us/pull/3247/files?diff=unified&w=0#diff-bed5f648b9858e5763ee90ad85315aac8a740a5e925a14a3a5ed9a5134afaea4R1-R13), [link](https://github.com/PolicyEngine/policyengine-us/pull/3247/files?diff=unified&w=0#diff-5559049c4f6b54ea1c2369578d50d1470e6eeb04100bd3daa371f16730ec7cfdR1-R23))
  - Add unit test files for the `mt_modified_income` and `mt_social_security_benefit` variables, with an example of a joint filer with various income sources and adjustments ([link](https://github.com/PolicyEngine/policyengine-us/pull/3247/files?diff=unified&w=0#diff-942ce5560eb27ebfcc64c0e101241e3f420e00f8f098a29d940574a567eec5e8R1-R15), [link](https://github.com/PolicyEngine/policyengine-us/pull/3247/files?diff=unified&w=0#diff-7bc2626d63157f936c0283c8eefe03ccecb23bed0161556268d8ba593b18efe1R1-R15))
* Add empty variable files for the Montana AGI additions, subtractions, and adjustments, which can be populated with specific items in the future ([link](https://github.com/PolicyEngine/policyengine-us/pull/3247/files?diff=unified&w=0#diff-7e89add23ed58dc2d2cf97162c607e8dfe9134325881f38f390771d6a1d20f4bR1-R14), [link](https://github.com/PolicyEngine/policyengine-us/pull/3247/files?diff=unified&w=0#diff-2b6675dd2e08d8c2183c20921a36399ebacc973595af37ccaefdc0073371d90aR1-R14), [link](https://github.com/PolicyEngine/policyengine-us/pull/3247/files?diff=unified&w=0#diff-6b6a4da3df46aa18ad78396ffe782cdbecfc3dd92de09705a8b4651ff8f375cdR1-R13))

### CODE REVIEW

File: changelog_entry.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:1]: delete extra line

File: policyengine_us/parameters/gov/states/mt/tax/income/subtractions/social_security/fraction/amount.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+values:
+  2021-01-01: 0.5
+
+metadata:
+  period: year
+  unit: /1
+  label: Montana social security subtraction total benefit fraction 1
+  reference:
+  - title: 2022 Montana Individual Income Tax Return Form, Taxable Social Security Benefits Schedule, Line 2
+    href: https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2023/05/Montana-Idividiual-Income-Tax-Return-Form-2-2022v6.2.pdf
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/mt/tax/income/subtractions/social_security/fraction/amount.yaml:11]: pages in link not in title

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/mt/tax/income/subtractions/social_security/fraction/amount.yaml:11]: all yaml

File: policyengine_us/variables/gov/states/mt/tax/income/subtractions/mt_agi_subtractions.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+from policyengine_us.model_api import *
+
+
+class mt_agi_subtractions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Montana AGI subtractions from federal AGI"
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/mt/tax/income/subtractions/mt_agi_subtractions.py:7]:
```suggestion
    label = "Montana subtractions from federal AGI"
```

File: policyengine_us/variables/gov/states/mt/tax/income/subtractions/mt_modified_income.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    value_type = float
+    entity = TaxUnit
+    label = "Montana modified income for the taxable social security benefits"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://mtrevenue.gov/wp-content/uploads/mdocs/form%202%202021.pdf" ""
+    )
+    defined_for = StateCode.MT
+    adds = [
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/mt/tax/income/subtractions/mt_modified_income.py:14]: make lists for each

File: policyengine_us/variables/gov/states/mt/tax/income/subtractions/social_security_benefit/mt_social_security_benefit.py [hunk 5]
```diff
@@ -0,0 +1,54 @@
+from policyengine_us.model_api import *
+
+
+class mt_social_security_benefit(Variable):
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/mt/tax/income/subtractions/social_security_benefit/mt_social_security_benefit.py:4]: make an eligibility file

File: policyengine_us/variables/gov/states/mt/tax/income/subtractions/mt_modified_income.py
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/mt/tax/income/subtractions/mt_modified_income.py:1]: add: 
`tax_exempt_dividend_income.py` in the same folder as the dividend income

Comment (PavelMakarchuk) [reply policyengine_us/variables/gov/states/mt/tax/income/subtractions/mt_modified_income.py:1]: to have a file in the additions

File: policyengine_us/variables/gov/states/mt/tax/income/subtractions/mt_modified_income.py [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+        "mt_agi_additions",
+        "tax_exempt_interest_income",
+        "mt_modified_income_benefit_fraction",
+        "student_loan_interest",
+    ]
+    subtracts = [
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/mt/tax/income/subtractions/mt_modified_income.py:20]: for subtractions: 
us_govt_interest

and file an issue for montana subtractions

File: policyengine_us/variables/gov/states/mt/tax/income/subtractions/mt_modified_income.py [hunk 7]
```diff
@@ -0,0 +1,25 @@
+from policyengine_us.model_api import *
+
+
+class mt_modified_income(Variable):
+    value_type = float
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/mt/tax/income/subtractions/mt_modified_income.py:5]: we can use `above_the_line_deductions` for page 6 line 7

**Review by ThapeloMasasa — COMMENTED** [review]
run 'make format', to remove extra lines

File: policyengine_us/parameters/gov/states/mt/tax/income/subtractions/social_security/excess_income/exceeding_income_fraction.yaml [hunk 8]
```diff
@@ -0,0 +1,15 @@
+description: Montana allows addition or subtraction from US AGI of the lesser or larger of US-taxable social security and an alternative amount with the complex calculation of the alternative subtraction amount including this fraction.
```
Comment (ThapeloMasasa) [inline policyengine_us/parameters/gov/states/mt/tax/income/subtractions/social_security/excess_income/exceeding_income_fraction.yaml:1]: The descriptions should be shorter and more precise

**Comment by qingqiuzhang** [comment]
@PavelMakarchuk Could you take time to review this PR? I'm not sure about the descriptions and labels in the parameter files.

File: policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/base_amount.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/base_amount.yaml:1]: this should be cap.yaml

File: policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/base_amount.yaml [hunk 9]
```diff
@@ -0,0 +1,24 @@
+description: Montana subtracts this amount from exceeding part of modified income (compared with income cap) depending on filing status.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/base_amount.yaml:1]:
```suggestion
description: Montana caps the taxable social security benefits at this amount of the modified gross income above the income threshold. 
```

File: policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/base_amount.yaml [hunk 10]
```diff
@@ -0,0 +1,24 @@
+description: Montana subtracts this amount from exceeding part of modified income (compared with income cap) depending on filing status. 
+metadata:
+  label: Montana social security subtraction exceeding income cap
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/base_amount.yaml:3]:
```suggestion
  label: Montana social security subtraction cap
```

File: policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/fraction/amount.yaml [hunk 11]
```diff
@@ -0,0 +1,15 @@
+description: Montana allows adjustments from US AGI of the lesser of US-taxable social security and an alternative amount with the complex calculation including this fraction which is applied to total social security benefits.
+values:
+  2021-01-01: 0.5
+
+metadata:
+  period: year
+  unit: /1
+  label: Montana social security subtraction total benefit fraction 1 # change later
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/fraction/amount.yaml:8]: need change

File: policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/fraction/amount.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/fraction/amount.yaml:1]: rename to income.yaml

File: policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/fraction/amount.yaml [hunk 12]
```diff
@@ -0,0 +1,15 @@
+description: Montana allows adjustments from US AGI of the lesser of US-taxable social security and an alternative amount with the complex calculation including this fraction which is applied to total social security benefits.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/fraction/amount.yaml:1]:
```suggestion
description: Montana multiplies the capped excess income by this fraction.
```

File: policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/fraction/benefit.yaml [hunk 13]
```diff
@@ -0,0 +1,15 @@
+description: Montana allows adjustments from US AGI of the lesser of US-taxable social security and an alternative amount with the complex calculation including this fraction which is applied to total social security benefits.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/fraction/benefit.yaml:1]:
```suggestion
description: Montana multiplies the smaller of the income cap fraction and the fraction of federal social security benefits.
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/fraction/benefit.yaml:1]: federal social security benefits is line 1 and 2 right?

File: policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/fraction/benefit.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/fraction/benefit.yaml:1]: rename to capped_income

File: policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/fraction/extra_income_fraction.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/fraction/extra_income_fraction.yaml:1]: rename to total_income.yaml

File: policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/fraction/extra_income_fraction.yaml [hunk 14]
```diff
@@ -0,0 +1,15 @@
+description: Montana allows adjustments from US AGI of the lesser of US-taxable social security and an alternative amount with the complex calculation including this fraction which is applied extra income amount.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/fraction/extra_income_fraction.yaml:1]:
```suggestion
description: Montana multiplies the sum of the capped income fraction and its product by this fraction.
```

File: policyengine_us/variables/gov/states/mt/tax/income/additions/mt_other_additions.py [hunk 15]
```diff
@@ -0,0 +1,14 @@
...
+    value_type = float
+    entity = TaxUnit
+    label = "Montana additions except addition to federal taxable social security benefits"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2023/05/Montana-Idividiual-Income-Tax-Return-Form-2-2022v6.2.pdf#page=4",
+    )
+    defined_for = StateCode.MT
+    adds = ["tax_exempt_dividend_income"]
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/mt/tax/income/additions/mt_other_additions.py:14]: make a list parameter for this. even if it includes only one file right now, might expand later

File: policyengine_us/variables/gov/states/mt/tax/income/adjustments/social_security_benefit/mt_social_security_benefits.py [hunk 16]
```diff
@@ -0,0 +1,57 @@
...
+        reduced_exceeding_income = max_(
+            exceeding_income - exceeding_income_cap, 0
+        )  # Line 13
+        capped_exceeding_income = min_(
+            exceeding_income, exceeding_income_cap
+        )  # Line 14
+
+        exceeding_income_fraction = (
+            p.excess_income.exceeding_income_fraction
+        )  # 0.5
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/mt/tax/income/adjustments/social_security_benefit/mt_social_security_benefits.py:35]: no specific parameter values in the comments as they might change

File: policyengine_us/variables/gov/states/mt/tax/income/subtractions/mt_other_subtractions.py [hunk 17]
```diff
@@ -0,0 +1,13 @@
...
+
+class mt_other_subtractions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Montana subtractions except subtraction from federal taxable social security benefits"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2023/05/Montana-Idividiual-Income-Tax-Return-Form-2-2022v6.2.pdf#page=5"
+    defined_for = StateCode.MT
+    adds = ["us_govt_interest"]
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/mt/tax/income/subtractions/mt_other_subtractions.py:12]: make a list parameter for this. even if it includes only one file right now, might expand later

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/fraction/benefit.yaml (hunk 13 above)
Comment (qingqiuzhang) [reply policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/fraction/benefit.yaml]: > federal social security benefits is line 1 and 2 right?
Yes, line 1 is the net amount of ss benefits and line 2 is a half of that.

Comment (qingqiuzhang) [reply policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/fraction/benefit.yaml]: > federal social security benefits is line 1 and 2 right?

Yes, line 1 is the net amount of ss benefits and line 2 is a half of that.

File: policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/base_amount.yaml [hunk 18]
```diff
@@ -0,0 +1,24 @@
...
+  period: year
+  unit: currency-USD
+  breakdown:
+  - filing_status
+  reference:
+  - title: Montana Code Annotated 2021 TITLE 15. TAXATION CHAPTER 30. INDIVIDUAL INCOME TAX Part 21. Rate and General Provisions (5)
+    href: https://leg.mt.gov/bills/mca/title_0150/chapter_0300/part_0210/section_0100/0150-0300-0210-0100.html
```
[the remark this answers is not in the document]
Comment (qingqiuzhang) [reply policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/base_amount.yaml]: > I think this was recently repealed, we need the newer version of the legal code

I just found MT posted the newer version at their website: https://www.leg.mt.gov/bills/mca/title_0150/chapter_0300/part_0210/section_0200/0150-0300-0210-0200.html
I think we'd be better use this.

**Review by MaxGhenis — CHANGES_REQUESTED** [review]
We already have variables for social security and dividends, and federal agi already includes taxable ss. Please use these and check the Montana logic

File: policyengine_us/variables/household/income/person/dividends/tax_exempt_dividend_income.py
Comment (PavelMakarchuk) [inline policyengine_us/variables/household/income/person/dividends/tax_exempt_dividend_income.py:1]: delete this file

**Comment by PavelMakarchuk** [comment]
> We already have variables for social security and dividends, and federal agi already includes taxable ss. Please use these and check the Montana logic

@MaxGhenis 
I believe that we have discussed in a call with @qingqiuzhang that we need to refactor the taxable_ss variable first as we need intermediate outputs for the Montana calculation and we agreed that we add `mt_social_security_benefits` and subtract `tax_unit_taxable_social_security` until we refactor the variable in a separate PR

Let me know if misunderstood

**Comment by MaxGhenis** [comment]
Social Security is a federal program; what does Montana Social Security benefits mean?

**Comment by PavelMakarchuk** [comment]
> Social Security is a federal program; what does Montana Social Security benefits mean?

The [tax form](https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2023/05/Montana-Idividiual-Income-Tax-Return-Form-2-2022v6.2.pdf#page=6) suggests that the excess of the Montana taxable social security over federal ss is considered an addition, while if the federal ss is greater than the Montana ss then this excess is considered a subtarction

image

**Comment by MaxGhenis** [comment]
[Legal code section 15-30-2101(21)(xiv)](https://www.leg.mt.gov/bills/mca/title_0150/chapter_0300/part_0210/section_0010/0150-0300-0210-0010.html) suggests Montana just taxes all Social Security benefits, compared to the 50%/85% federal taxability. The math in the form looks like it reproduces that to account for residents who might not have received those benefits entirely in Montana.

If that's right then you can just add social_security to the additions and taxable_social_security to the subtractions.

I was thrown by `mt_social_security_benefits` - if we needed this variable (and I don't think we do), it should be named `mt_taxable_social_security`.

**Comment by PavelMakarchuk** [comment]
> > Social Security is a federal program; what does Montana Social Security benefits mean?
> 
> The [tax form](https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2023/05/Montana-Idividiual-Income-Tax-Return-Form-2-2022v6.2.pdf#page=6) suggests that the excess of the Montana taxable social security over federal ss is considered an addition, while if the federal ss is greater than the Montana ss then this excess is considered a subtarction
> 
> image

w

> [Legal code section 15-30-2101(21)(xiv)](https://www.leg.mt.gov/bills/mca/title_0150/chapter_0300/part_0210/section_0010/0150-0300-0210-0010.html) suggests Montana just taxes all Social Security benefits, compared to the 50%/85% federal taxability. The math in the form looks like it reproduces that to account for residents who might not have received those benefits entirely in Montana.
> 
> If that's right then you can just add social_security to the additions and taxable_social_security to the subtractions.
> 
> I was thrown by `mt_social_security_benefits` - if we needed this variable (and I don't think we do), it should be named `mt_taxable_social_security`.

Thank you for clarifying, we will review the law one more time but I think that is right

File: policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/base_amount.yaml (hunk 18 above)
[the remark this answers is not in the document]
Comment (MaxGhenis) [reply policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/base_amount.yaml]: is this resolved?

**Review by MaxGhenis — APPROVED** [review]
Good to merge if outstanding comment threads are resolved.

File: policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/base_amount.yaml (hunk 18 above)
[the remark this answers is not in the document]
Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/mt/tax/income/adjustments/social_security/base_amount.yaml]: Yes, we did

**Pull request merged**